### ISSUE TITLE
upload new sampling data: nilla.au 2016 (year 0)

### ISSUE THREAD

**Issue by nutnetadmin** [nutnetadmin/nutnet-data-tracker#10]
received 15-Feb-2017

**Comment by nutnetadmin** [comment]
started uploading this but found discrepancy in subplot for Block 2, Plot 14. Emailed JA Catford on 15 March 2018 to check: 

In 2016, biomass is recorded for subplot B, but cover and PAR are recorded for subplot D. In the plan, the core subplot is listed as subplot B.
In 2017, the subplot for mass is not recorded (should I assume D?).  Cover and PAR are again recorded for subplot D. And this time, in the plan, the core subplot is listed as subplot D.

**Comment by nutnetadmin** [comment]
JA Catford confirms that all measurements for block 2 plot 14 come from subplot D. Email from 
19-March-2018.

14D is the subplot that I have sampled throughout. 
I initially randomly selected subplot B but my dad had recently dug a hole in that subplot (as it happened… - I..e before I marked out all of the plots) so I changed it to subplot D before any sampling was done. I clearly forgot to update all of the spreadsheets before submitting the 2016 data.  

I can confirm that all measurements (including biomass) were taken from 14D in both 2016 and 2017.
